M2/INSTALL-CMake.md

```diff
@@ -175,6 +175,7 @@ For a complete list, along with descriptions, try `cmake -LAH .` or see `cmake/c
 The following flags can be also set by giving environment variables to CMake (e.g. `errorDepth=2 cmake .`), though setting them this way is ephemeral.
 - `CheckDocumentation:STRING=true`: check documentation for completeness
 - `IgnoreExampleErrors:STRING=true`: ignore errors in example code
+- `MakeDocumenation:STRING=true`: make the documentation
```

Comment:
Oops, typo!  I'll fix this on the next push.

Reply:
This is not intended for porting all options of `installPackage` to CMake..